README.md

```diff
@@ -197,6 +197,16 @@ gamebot cancel
 Victor Barna and Deng Yaping canceled a challenge against Wang Hoe and Zhang Jike.
 ```
 
+####  gamebot taunt &lt;opponent&gt; ... [with &lt;teammate&gt; ...] 
+
+Taunt another player.
+
+```
+gamebot taunt @WangHoe
+
+Victor Barna said Wang Hoe sucks at ping pong!
```

Comment:
I think it should be "says", because this is happening right now. Victor Barna just sucks at ping pong. It's not going to change.

Reply:
This is incorrect, the code says "says".